action.yaml

```
@@ -6,52 +6,78 @@ description:
+      This is some unique verbatim subset of the comment that is to be used to
+      determine if a comment has already been created against the PR that
+      instructs the author how to resolve the given problem.
+    default: created by action-conditional-pr-comment
+  add-or-remove-comment:
     required: true
     description:
-      Whether or not to delete the comment that has been added to the PR upon
-      problem resolution.
-    default: "true"
+      true indicates the comment is to be added/updated to/within the PR, false
+      indicates the comment is to be removed from the PR.
   github-token:
     required: true
     description:
-      GitHub token that can create/delete comments. e.g. 'secrets.GITHUB_TOKEN'
+      GitHub token that can add/update/delete comments. e.g.
+      'secrets.GITHUB_TOKEN'
+  comment-author:
+    required: false
+    description: The author of the comment upon addition.
+    default: open-turo-bot
 runs:
   using: composite
   steps:
-    - name:
-        Check to see if problem resolution instructions comment already present
-        on the PR
-      uses: peter-evans/find-comment@v2
+    - name: Lets see if we can get the PR number from the github context
+      shell: bash
+      run: echo "PR ${{ github.event.pull_request.number }}"
+    - name: Check to see if comment already present on the PR
       if: github.event_name == 'pull_request'
       id: fc
+      uses: peter-evans/find-comment@v2
       with:
-        issue-number: ${{ github.event.pull_request.number }}
-        comment-author: "github-actions[bot]"
-        body-includes: ${{ inputs.instruction-text-detector }}
-    - name: Conditionally create instructions comment for PR author
+        issue-number: ${{ inputs.pull-request-number }}
+        comment-author: ${{ inputs.comment-author }}
+        body-includes: ${{ inputs.text-detector }}
+        token: ${{ inputs.github-token }}
+    - name: Conditionally add comment to PR
       if:
-        github.event_name == 'pull_request' && failure() &&
-        steps.fc.outputs.comment-id == ''
+        github.event_name == 'pull_request' && inputs.add-or-remove-comment ==
+        'true' && steps.fc.outputs.comment-id == ''
       uses: peter-evans/create-or-update-comment@v2
       with:
-        issue-number: ${{ github.event.pull_request.number }}
-        body: ${{ inputs.instruction-text}}
-    - name: Delete instructions comment from PR if problem has been resolved
+        issue-number: ${{ inputs.pull-request-number }}
+        ## Place the comment detection text within an HTML comment.
+        body:
+          ${{ format('{0} <!-- {1} -->', inputs.comment, inputs.text-detector)}}
+        token: ${{ inputs.github-token }}
+        reactions: eyes
+    - name: Conditionally update existing comment within PR
+      if:
+        github.event_name == 'pull_request' && inputs.add-or-remove-comment ==
+        'true' && steps.fc.outputs.comment-id != ''
```

```suggestion
        github.event_name == 'pull_request' 
        && inputs.add-or-remove-comment == 'true' 
        && steps.fc.outputs.comment-id != ''
```
Was this manual wrapping?  If it could be wrapped on the operator it makes it easier to read.

---

No, I did not manually use that style. Will see if I can tweak it though and not have prettier or whatever change the style.

---

Sadly, trying to use a more human friendly style to read, pre-commit just changes the style back to what you see in the PR.